CONTRIBUTING.md

```diff
@@ -418,22 +420,23 @@ For example, the image referenced above actually lives in `superset-frontend/ima
 
 #### OS Dependencies
 
-Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.
+Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.  
+You also need to install MySQL or [MariaDB](https://mariadb.com/downloads).
```

Comment:
@craig-rueda / @willbarrett is this line accurate, or are the others that we should mention support for here?

Reply:
Ideally folks would also install Postgres. Superset works with both metadata DBs.

Reply:
~~Oh, that might well be required in the tests as well (I suppose due to [`psycopg`](https://github.com/apache/superset/blob/master/requirements/development.txt#L26)).~~ 
Actually, I tried to install Superset (1.1.0) on a _blank_ F32 VM (`rpm -qa | grep postgres` returns empty) alongside its test dependencies (cf. #14052 ) and, up to running the tests themselves (`$ tox`), no issue was raised (unlike for MySQL). 

Do we agree we only want to mention these OS dependencies in the “set up your system to run the tests” (and not in the [install from scratch](https://github.com/apache/superset/blob/master/docs/src/pages/docs/installation/installing_scratch.mdx) docs), as we wouldn't want users to install both RDBMS by default?

Reply:
@willbarrett & @rusackas : it looks like `psycopg2-binary` [doesn't require](https://www.psycopg.org/docs/install.html) an explicit PG install, while `mysqlclient` [does have](https://pypi.org/project/mysqlclient/) OS dependencies.

So çaybe would it be better to refer to `mysqlclient` page for installs (where both `brew` and `apt`/`yum` methods are detailed… and hopefully maintained), rather than to vanilla MySQL/MariaDB pages; what do you think?